hack/os-packages/Makefile

```diff
@@ -1,17 +1,20 @@
 RPM_PACKAGES_GETTER_IMAGE = mesosphere/rpm-artifacts-getter
-RPM_FIPS_PACKAGES_GETTER_IMAGE = mesosphere/rpm-artifacts-getter-fips
-
+CENTOS_BASE_IMAGE=mesosphere/centos:7.9.2009.minimal
```

Comment:
hardcoding to centos:7.9.2009.minimal for now. this will be parameter in future iterations.